packages/theme/README.md

```diff
@@ -15,6 +15,42 @@ npm install @spectrum-web-components/themes
 yarn add @spectrum-web-components/themes
 ```
 
+## Quick start
```

Comment:
This part is very new for the usage of this package, please take a closer look for whether it makes sense to you, too.